hs_core/hydroshare/resource.py

```diff
@@ -960,6 +958,8 @@ def publish_resource(user, pk):
     """
     resource = utils.get_resource_by_shortkey(pk)
 
+    # TODO: whether a resource can be published is not considered in can_be_published
+    # TODO: can_be_published is currently an alias for can_be_public_or_discoverable
```

Comment:
@alvacouch I believe can_be_published() has the same requirement as can_be_public_or_discoverable, at least at this point. Not sure whether stricter requirement would be added for whether a resource can be published or not in the future, but it makes sense to me that if a resource can be made public or discoverable, this resource should be publishable as well. Looping in @dtarb and @horsburgh for their comments as well.